lib/src/main/java/com/auth0/android/lock/views/ValidatedInputView.java

```diff
@@ -95,6 +96,7 @@
         int DATE = 7;
         int TEXT_NAME = 8;
         int NUMBER = 9;
+        int SSO_USERNAME = 10;
```

Comment:
I am not buying this new name, its kind of cryptic. Ideally it's just a non-empty validation why we need to have a `SSO_USERNAME`? The style should be the same as the username one, and it's still a username in a sense